src/Breadcrumb.js

```diff
@@ -0,0 +1,47 @@
+import React, { cloneElement } from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import ValidComponentChildren from './utils/ValidComponentChildren';
+
+const Breadcrumb = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    /**
+     * CSS classes for the wrapper `ol` element
+     */
+    className: React.PropTypes.string,
+    /**
+     * HTML id for the wrapper `ol` element
+     */
+    id: React.PropTypes.string
+  },
+
+  getDefaultProps() {
+    return {
+      bsClass: 'breadcrumb'
+    };
+  },
+
+  render() {
+    const classes = this.getBsClassSet();
+
+    return (
+      <ol role="navigation" aria-label="breadcrumbs" className={classNames(this.props.className, classes)}>
```

Comment:
I forgot to pass them into the `ol` element. I fixed it and tested if the `id` is set correctly in the `ol` element.